src/Compilers/CSharp/Portable/Compiler/MethodBodySynthesizer.cs

```diff
@@ -102,7 +102,7 @@ internal static class MethodBodySynthesizer
                         objectType)
                     { WasCompilerGenerated = true },
                     thisReference,
-                    RefKind.None,
+                    false,
```

Comment:
>false [](start = 20, length = 5)

A named argument would be nice.

Reply:
Unfortunately this is one of those rare cases where adding named arguments actually makes overload resolution ambiguous.